src/wp-login.php

```diff
@@ -214,11 +214,14 @@ function login_footer($input_id = '') {
 
 	// Don't allow interim logins to navigate away from the page.
 	if ( ! $interim_login ): ?>
-	<p id="backtoblog"><a href="<?php echo esc_url( home_url( '/' ) ); ?>"><?php
-		/* translators: %s: site title */
-		printf( _x( '&larr; Back to %s', 'site' ), get_bloginfo( 'title', 'display' ) );
-	?></a></p>
-	<?php the_privacy_policy_link( '<div class="privacy-policy-page-link">', '</div>' ); ?>
+	<p id="backtoblog" class="visit-site-link"><a href="<?php echo esc_url( home_url( '/' ) ); ?>"><?php
+			/* translators: %s: site title */
+			printf( _x( 'Visit <strong>%s</strong>', 'site' ), get_bloginfo( 'title', 'display' ) );
+			?></a></p>
```

Comment:
Indentation fix to match the surrounding code:

```suggestion
	<p id="backtoblog" class="visit-site-link"><a href="<?php echo esc_url( home_url( '/' ) ); ?>"><?php
		/* translators: %s: site title */
		printf( _x( 'Visit <strong>%s</strong>', 'site' ), get_bloginfo( 'title', 'display' ) );
	?></a></p>
```